src/Components/People.js

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios'
+import React, { Component } from 'react'
+
+export class People extends Component {
+    constructor(){
+        super()
+
+        this.state = {
+            input: "",
+            people: [],
+            name: "",
+            age: "",
+            gender: "",
+            isError: false,
+        }
+    }
+
+    componentDidMount () {
+        this.getPeople()
+    }
+
+    getPeople = async () => {
+        const {data} = await axios.get(`https://ghibliapi.herokuapp.com/people`)
+        // console.log(data)
+
+        this.setState ({
+            people: data,
+        })
+    }
+
+    handleInput = (event) => {
+        this.setState({
+            input: event.target.value,
+        })
+    }
+
+    handleSubmit = async (event) => {
+        event.preventDefault()
+
+        const {input,people} = this.state
+        const person = people.filter((p) => p.name === input)
+        // console.log(person[0])
+
+        if(person[0]) {
+            this.setState({
+                name: person[0].name,
+                age: person[0].age,
+                gender: person[0].gender,
+                isError: false,
+            })
+        } else {
+            this.setState ({
+                name: "",
+                age: "",
+                gender: "",
+                isError: !this.state.isError,
```

Comment:
This seems dangerous! what would happen if i search for two non-existent people in a row?

Instead, just set isError to true!